x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/pipelines/ml_inference/configure_pipeline.tsx

```diff
@@ -93,12 +94,14 @@ export const ConfigurePipeline: React.FC = () => {
       ),
       value: PIPELINE_SELECT_PLACEHOLDER_VALUE,
     },
-    ...(existingInferencePipelines?.map((pipeline) => ({
-      disabled: pipeline.disabled,
-      dropdownDisplay: <PipelineSelectOption pipeline={pipeline} />,
-      inputDisplay: pipeline.pipelineName,
-      value: pipeline.pipelineName,
-    })) ?? []),
+    ...(existingInferencePipelines
+      ?.filter((pipeline) => !filterModelId.includes(pipeline.modelId))
```

Comment:
There is already a selector in the `ml_inference_logic` for building this list of pipelines. We should do this filtering there instead of in this component.